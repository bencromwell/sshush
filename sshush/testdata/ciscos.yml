ciscos:
  Config:
    Ciphers: aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    KexAlgorithms: +diffie-hellman-group1-sha1
    HostKeyAlgorithms: ssh-rsa,ssh-dss
    PubkeyAuthentication: "no"
  Hosts:
    - fooas*.adm
    - foocs*.adm
    - foocr01.adm
    - cs*.foo.adm
