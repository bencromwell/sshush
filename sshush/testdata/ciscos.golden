# Generated by sshush v0.0.0-dev
# From testdata/ciscos.yml

# ciscos
Host cs*.foo.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

Host fooas*.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

Host foocr01.adm
    HostName foocr01.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

Host foocs*.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no
